package.json

```diff
@@ -18,7 +18,7 @@
     "clean:package_clean": "npx lerna run clean --parallel",
     "clean:lerna": "npx lerna clean --yes ",
     "clean:docs": "npx del-cli dist",
-    "bootstrap": "npx lerna bootstrap",
+    "bootstrap": "npx lerna bootstrap -- --force",
```

Comment:
Yep, this is need in order to pass the force arg.
By the way, we need it because when using npm >=7 it tries to install the peer dependencies and it block installations if an upstream dependency conflict is present that cannot be automatically resolved.